socorro/tests/processor/rules/test_mozilla.py

```diff
@@ -791,6 +792,58 @@ def test_absent_seconds_since_last_crash(self, tmp_path):
         assert status.notes == []
 
 
+class TestMacBootArgsRule:
+    @pytest.mark.parametrize(
+        "processed, expected",
+        [
+            ({}, False),
+            ({"json_dump": {}}, False),
+            ({"json_dump": {"mac_boot_args": None}}, False),
+            ({"json_dump": {"mac_boot_args": ""}}, False),
```

Comment:
seems like it is covered by downstream tests, but I feel like this test should still cover at least one truthy case:
```suggestion
            ({"json_dump": {"mac_boot_args": ""}}, False),
            ({"json_dump": {"mac_boot_args": "-v"}}, True),
```

Reply:
I can add that. :+1: